static/app/views/issueDetails/streamline/eventGraph.tsx

```diff
@@ -591,21 +593,27 @@ const SummaryContainer = styled('div')`
   }
 `;
 
-const Callout = styled(Button)<{isActive: boolean}>`
-  cursor: ${p => (p.isActive ? 'initial' : 'pointer')};
-  border: 1px solid ${p => (p.isActive ? p.theme.purple100 : 'transparent')};
-  background: ${p => (p.isActive ? p.theme.purple100 : 'transparent')};
-  padding: ${space(0.5)} ${space(2)};
-  box-shadow: none;
-  height: unset;
-  overflow: hidden;
-  &:disabled {
-    opacity: 1;
-  }
-  &:hover {
+const CalloutButton = withChonk(
+  styled(Button)<{isActive: boolean}>`
+    cursor: ${p => (p.isActive ? 'initial' : 'pointer')};
     border: 1px solid ${p => (p.isActive ? p.theme.purple100 : 'transparent')};
-  }
-`;
+    background: ${p => (p.isActive ? p.theme.purple100 : 'transparent')};
+    padding: ${space(0.5)} ${space(2)};
+    box-shadow: none;
+    height: unset;
+    overflow: hidden;
+    &:disabled {
+      opacity: 1;
+    }
+    &:hover {
+      border: 1px solid ${p => (p.isActive ? p.theme.purple100 : 'transparent')};
+    }
+  `,
+  styled(Button)<never>`
```

Comment:
the never type here is because the button props are a union type of ButtonPropsWithAriaLabel and ButtonPropsWithoutAriaLabel, meaning we cannot infer them correctly using our helper, but since we don't need all of them, we can bypass it all by setting the type to never